### ISSUE TITLE
Again my followed ZeroTalk topics was silently removed and i am following next to nothing now

### ISSUE THREAD

**Issue by slrslr** [HelloZeroNet/ZeroNet#2007]
### Step 1: Please describe your environment

  * ZeroNet version: 3863
  * Operating system: Windows
  * Web browser: FF
  * Tor status: available
  * Opened port: no
  * Special configuration: multiuser_local ; using zeroid & kxoid

### Step 2: Describe the problem:

This happened to me already and today i found out again that somehow i am not following important ZeroTalk topics. So i spend some time to discover that almost no zerotalk topic of mine i am following (maybe only a few topics i re-followed).

recently i tried to create few kxoid topics on zerotalk, unsure if it can be related. I am using same old identity masterseed on zerohello.

So this is a big problem to me, let me know steps needed to discover cause of this bug and how to fix this.

**Comment by slrslr** [comment]
on my ZeroMe profile i also see i am NOT following majority of my posts and i do not think it is because i unsubscribed. On site, the option "Follow comments on your posts" is ticked.

**Comment by HelloZeroNet** [comment]
Do you see any error in zerohello stats section? (shows by clicking on "in x ms from x sites" link)
There was a fix in rev3864 please update and try again.

**Comment by slrslr** [comment]
yes, i am already using rev3864 (sorry, indicated wrong version in leading post).
i do not see any error on /1HeLLo4uzjaLetFx6NH3PMwFP3qbRbTf3D/?Stats (except it do not load any data transfer in the graph). In ZeroHello newsfeed also no error when clicked "from X sites in Xs"

**Comment by krixano** [comment]
![image](https://user-images.githubusercontent.com/4098042/57040096-147efd80-6c24-11e9-9787-e1da5285d5df.png)

**Comment by HelloZeroNet** [comment]
@krixano Yes this can be a problem, but the topic starter issue is about following topics, that should independent from the username.

**Comment by slrslr** [comment]
Maybe the cause of this issue is not a Zeronet bug, but the issue where i had to recover my old zeronet profile (users.json ?) and so i have lost substantial amount of the followed topics. I will update if i will find that it happened again and mentioned can not be true.
